app/migrations/Version20240205082236.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240205082236 extends LoggableMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add active field to Administrator';
+    }
+
+    public function preUp(Schema $schema): void
+    {
+        $this->addSql('UPDATE Administrators SET email = "noreplay@irontec.com" WHERE id = 1 AND email IS NULL');
+        parent::preUp($schema); // TODO: Change the autogenerated stub
```

Comment:
Remove this line, parent::preUp does nothing

Reply:
Line removed.